daal4py/sklearn/ensemble/GBTDAAL.py

```diff
@@ -128,9 +134,14 @@ def _check_params(self):
     def _more_tags(self):
         return {"allow_nan": self.allow_nan_}
 
+    def __sklearn_tags__(self):
```

Comment:
I think this be behind a sklearn_check_version.

Reply:
Done.